pom.xml

```diff
@@ -0,0 +1,81 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>hzdocs</groupId>
+    <artifactId>hzdocs</artifactId>
+    <version>0.1-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>${project.artifactId}</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.release>17</maven.compiler.release>
+        <version.hazelcast>5.5.0</version.hazelcast>
```

Comment:
Can we keep this in sync with the Hazelcast version?
E.G. this _should_ be `6.0.0-SNAPSHOT` in `main`.

Reply:
This is a placeholder. I'll add a comment to explain it.

Reply:
Done in #1283. Further updates will be made there.